README.md

```diff
@@ -1,8 +1,26 @@
-# ESP-BSP
-Board support packages for Espressif development boards.
+# ESP-BSP: Espressif's Board Support Packages
+Board support packages for development boards using Espressif's SoCs, written in C.
+
+## Supported boards
+| Board name | SoC | Features | Photo |
+|---|---|---|---|
+| ESP-WROVER-KIT | ESP32 | LCD display, uSD card slot | <img src="docu/pics/wrover.jpg" width="150"> |
+| ESP-BOX | ESP32-S3 | LCD display with touch, audio codec + power amplifier,<br>accelerometer and gyroscope | <img src="docu/pics/box.webp" width="150"> |
+| ESP32-Azure IoT Kit | ESP32 | OLED display, uSD card slot, accelerometer,<br>magnetometer, humidity, pressure, light<br>and temperature sensors | <img src="docu/pics/azure.webp" width="150"> |
+| ESP32-S2-Kaluga Kit | ESP32-S2 | LCD display, audio codec + power amplifier,<br>smart LED and camera | <img src="docu/pics/kaluga.webp" width="150">  |
+
+## How to use
+
+Best way to start with ESP-BSP is trying one of the [examples](examples) on your board. Every example contains `README.md` with list of supported boards.
```

Comment:
> I would like see table with examples (with links) and with info about supported boards.

Good point. I'll add it.

> it is not really good, because some example can be for two board, some for all boards.

Yeah, I'd like to avoid copy-pasting code, if it is possible. I'll try to implement a feature that would allow as to change the board from menuconfig. If it was possible, I'd prefer to have all examples in one `examples` folder (and not in board-specific folders).

Reply:
> Very nice Readme. It is great. But here, I would like see table with examples (with links) and with info about supported boards.
> 
> As I said yesterday, I wanted to put each example into separate folder with name of the board, it is not really good, because some example can be for two board, some for all boards.

Directly linked examples are good idea, agree! Not added yet.